controls/radgauge/getting-started/creating-numeric-gauge.md

```diff
@@ -10,54 +10,44 @@ position: 2
 
 # Creating Numeric Gauge
 
-One of the gauges you can create with the __RadGauge__ control, is a numeric gauge. It is represented by a container with a numerical scale in it. The numerical scale uses a numerical indicator to display the desired value. This topic will guide you in creating a trivial numeric gauge and will explain the following:
-
-* [Defining a Numeric Gauge](#defining-a-numeric-gauge)
-
-* [Defining a NumericScale](#defining-a-numericscale)
+One of the gauges you can create with the __RadGauge__ control, is a numeric gauge. It is represented by a container with a numerical scale in it. The numerical scale uses a numerical indicator to display the desired value. This topic will walk you through the creation of a sample application that contains a numeric RadGauge.
 
+* [Assembly References](#assembly-references)
+* [Defining a Numeric Gauge with a NumericScale](#defining-a-numeric-gauge-with-a-numericscale)
 * [Defining a Numeric Indicator](#defining-a-numeric-indicator)
-
 * [Set Indicator's Position](#set-indicators-position)
 
-* [Configuring the Gauge](#configuring-the-gauge)
+## Assembly References
+
+In order to use the __RadGauge__ control in your projects, you have to add references to the following assemblies:			
 
-## Defining a Numeric Gauge
+* __Telerik.Windows.Controls__
+* __Telerik.Windows.Controls.DataVisualization__
 
->In order to use the __RadGauge__ control and its components you have to reference the following assemblies:
->	- __Telerik.Windows.Controls.DataVisualization.dll__
->	- __Telerik.Windows.Controls.dll__
+## Defining a Numeric Gauge with a NumericScale
 
-There is no special container for the numeric scale, as for the radial or linear scales. However you can use any of the 9 radial gauge and 2 linear gauge controls to wrap the __NumericScale__ control. Additionally you can put NumericScale to the Grid container too.
+There is no special container for the numeric scale, as for the radial or linear scales. However you can use any of the 9 radial gauge and 2 linear gauge controls to wrap the __NumericScale__ control. Additionally you can put a NumericScale in a Grid container too. In order to define a numerical scale inside the __Gauge__ control you have to use the __NumericScale__ control. The __Gauge__ works as an __ItemsControl__, which takes __NumericScale__ controls as items. This means that you can have more than one scale inside the gauge.

```

Comment:
There is no special container for the numeric scale, as for the radial or linear scales. => Unlike the radial and linear scales, the numeric scale doesn't have (or require? not sure what is more appropriate) a special container.
Also, add links to the radial and linear scales, for easier orientation and navigation.

Additionally you can put a NumericScale in a Grid container too. => Additionally, you can put a __NumericScale__ in a __Grid__ container too. In order to define a numerical scale inside the __Gauge__ control, you have to...